kalamine/cli.py

```diff
@@ -178,9 +178,14 @@ def create(output_file: Path, geometry: str, altgr: bool, odk: bool) -> None:
 
 @cli.command()
 @click.argument("filepath", nargs=1, type=click.Path(exists=True, path_type=Path))
-def watch(filepath: Path) -> None:
+@click.option(
+    "-a", "--angle-mod/--no-angle-mod",
+    default=False,
+    help="Apply Angle-Mod (which is a [ZXCVB] permutation with the LSGT key (a.k.a. ISO key))",
```

Comment:
This is consistent with the rest but it doesn’t fit in the quick help : 

```
$ kalamine
Usage: kalamine [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  create   Create a new TOML layout description.
  guide    Show user guide and exit.
  make     Convert TOML/YAML descriptions into OS-specific keyboard drivers.
  version  Show version number and exit.
  watch    Watch a TOML/YAML layout description and display it in a web...
```

Maybe a string constant could be used here (DRY ?).